app/utils/crypto/symmetric/recrypt.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class Crypto::Symmetric::Recrypt
+
+  def initialize(current_user, team, private_key)
+    @current_user = current_user
+    @team = team
+    @private_key = private_key
+
+    perform
```

Comment:
call perform from the outside, not inside constructor ...
BlaClass.new(args).perform